libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -3073,14 +3073,16 @@ int WaveTrack::GetNumClips() const
 int WaveTrack::GetNumClips(double t0, double t1) const
 {
    const auto clips = SortedClipArray();
-   const auto firstIn =
-      std::find_if(clips.begin(), clips.end(), [t0](const auto& clip) {
-         return t0 < clip->GetPlayEndTime();
+   // Find first position where the comparison is false
+   const auto firstIn = std::lower_bound(clips.begin(), clips.end(), t0,
```

Comment:
I prefer the way it was before: easier to understand (I always have to look up the subtlety between lower_bound and upper_bound) and on usage average seldom more expansive performance-wise. But sure ...